test/linalg/qr.jl

```diff
@@ -144,3 +165,10 @@ let
     Q=full(qrfact(A)[:Q])
     @test vecnorm(A-Q) < eps()
 end
+
+qrpa = qrfact(areal,Val{true})
+qrpa = convert(QRPivoted{BigFloat},qrpa)
+#@test A_ldiv_B!(qrpa,zeros(BigFloat,n)) ≈ zeros(BigFloat,n)
```

Comment:
Why are these commented out? Should they be deleted or should they have a comment explaining why they are not run?